main.go

```diff
@@ -4,6 +4,8 @@ package main
 
 import "fyne.io/fyne/v2/app"
 
+// see the readme for installation instructions
```

Comment:
This documentation is not well formed, is it really needed when a README file is right next to it?

Reply:
I think this is a strange place to put installation documentation. I think the README will be good enough.

Reply:
Yeah, this is a matter of taste. In my experience people skip documentation. 

But i know i’m one of the few people that works this way. You probably want to ditch it.

Reply:
Our projects try to keep full sentences for documentation. Our guidelines are at https://github.com/fyne-io/fyne/wiki/Contributing